src/javascript/ContentEditor.redux.jsx

```diff
@@ -38,7 +65,8 @@ const ContentEditorReduxCmp = ({client, mode, uuid, lang, uilang, site, contentT
         unregisterListeners: () => {
             unRegisterBlockListener();
         },
-        shouldRedirectBeadCrumb: () => false
+        shouldRedirectBeadCrumb: () => false,
+        renameNode: rename
```

Comment:
rename the **renameNode** by something more meaningful like: **handleRenamedNode** or something else